robot/robot.py

```diff
@@ -1,13 +1,16 @@
 from pathlib import Path
 import time
+from typing import List, Union
 
 from robot.board import BoardList
 from robot.camera import Camera
-from robot.game import GameState
+from robot.game import GameState, GameMode
 from robot.motor import MotorBoard
 from robot.power import PowerBoard
 from robot.servo import ServoBoard
 
+Board = Union[Camera, GameState, MotorBoard, PowerBoard, ServoBoard]
```

Comment:
Do we semantically want this rather than using the base class? I.e. even in the presence of new subclasses, `Board` is closed to these five specifically?

Reply:
I originally used the base class, but for some reason it didn't like it. I should certainly have another play at some point